andela/ah-kgl-avengers-frontend
# #165020164 User Login

### What does this PR do?
User login
### Description of Task to be completed?
- create a user login interface
- add user login validation
- login a user
### How should this be manually tested?
 clone https://github.com/andela/ah-kgl-avengers-backend.git
     - git checkout ft-user-login-165020164
    - npm test
    - http://localhost:8000/login {
       - login with email: fridolinho@gmail.com, password: qwerty123
    }
   - check validation with wrong/ empty input
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[User login](https://www.pivotaltracker.com/story/show/165020164)
### Screenshots (if appropriate)
N/A
<img width="1230" alt="image" src="https://user-images.githubusercontent.com/18733531/59496700-4d042f00-8e92-11e9-98c3-f25a4b96845f.png">

### Questions:
N/A
